sp_mock/frontend/src/App.vue

```diff
@@ -1,5 +1,17 @@
 <script setup>
 import { RouterView } from "vue-router";
+import layer8_interceptor from 'layer8_interceptor'
+
+try{
+  layer8_interceptor.initEncryptedTunnel({
+    ServiceProviderURL: 'localhost:8000', // TODO: this needs to become the same form as the VITE_BACKEND_URL
+    Layer8Scheme: 'http',
+    Layer8Host: 'localhost',
+    Layer8Port: '5001'
```

Comment:
**[reminder]** we can abstract this out and make it optional. In dev mode, clients (devs in this case) can specify the proxy URL but in prod mode (the default mode), we can specify this internally so that the clients only specify their backend

Reply:
Yes I agree. I'm going to make this a separate issue and assign it to myself. Though, I wonder if we should make the dev mode the default.... otherwise i imagine us getting spammed with however many erroneous requests that simply aren't configured appropriately.

Reply:
Officially issue #88